pkg/resources/cluster/controller/controller-deployment.yaml

```diff
@@ -18,6 +18,37 @@ spec:
       labels:
         app.kubernetes.io/component: linstor-controller
     spec:
+      initContainers:
+        - name: run-migration
+          image: linstor-controller
+          args:
+            - runMigration
+          env:
+            - name: K8S_AWAIT_ELECTION_ENABLED
+              value: "1"
+            - name: K8S_AWAIT_ELECTION_NAME
+              value: "linstor-controller"
+            - name: "K8S_AWAIT_ELECTION_LOCK_NAME"
+              value: "linstor-controller"
+            - name: "K8S_AWAIT_ELECTION_LOCK_NAMESPACE"
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: K8S_AWAIT_ELECTION_IDENTITY
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+          securityContext:
+            readOnlyRootFilesystem: true
+            capabilities:
+              drop:
+                - ALL
+          volumeMounts:
```

Comment:
The container references `/var/log/linstor-controller`. Does this need to be mounted?

Reply:
No, the `linstor-config` tool only logs to stdout/stderr.